apps/site/.eslintrc.cjs

```diff
@@ -29,6 +29,11 @@ module.exports = {
             importNames: ["FC", "VFC", "VoidFunctionComponent"],
             message: "Please use FunctionComponent instead.",
           },
+          {
+            name: "framer-motion",
+            importNames: ["motion"],
+            message: "Please use 'm' instead.",
```

Comment:
Can we explain why please?

Reply:
Updated it as `Please use 'm' instead for 'framer-motion' to lazy-load correctly.`